README.md

```diff
@@ -1,13 +1,34 @@
+![commercetools-icon](https://github.com/commercetools/press-kit/blob/master/PNG/72DPI/CT%20logo%20horizontal%20RGB%2072dpi.png?raw=true)
+
 # eslint-config-commercetools
 
+[![David Dependencies Status][david-icon]][david]
+[![Travis Build Status][travis-icon]][travis]
```

Comment:
Good question, not sure what the build will do though 💭 . I will remove it for now and add it back if need be